core/src/main/scala/org/http4s/Header.scala

```diff
@@ -120,6 +120,17 @@ object Header {
     }
   }
 
+  /** Helper trait that provides a default way of rendering the value provided a Renderer */
+  trait RecurringRenderer extends Recurring {
+    type Value
+    implicit val renderer: Renderer[Value]
```

Comment:
Any ideas on how to initialize this by default? My attempts ended poorly 😄

Reply:
Eager abstract values make me nervous. How about a def?

Reply:
sure